src/Features/LanguageServer/Protocol/Handler/RequestExecutionQueue.RequestMetrics.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler
+{
+    internal partial class RequestExecutionQueue
+    {
+        internal class RequestMetrics
+        {
+            private readonly string _methodName;
+            private readonly SharedStopwatch _sharedStopWatch;
+            private TimeSpan? _queuedDuration;
```

Comment:
_queuedDuration?  or _queueDuration?  If the former... can you clarify what this means?

Reply:
_queuedDuration, i.e. the amount of time that the request was queued for (time spent in queue)